tasks/binary.yml

```diff
@@ -9,4 +9,4 @@
     group: 'root'
     mode: '0755'
   notify: [ 'Restart NeoFS Storage' ]
-  when: not (neofs_storage__use_compose | bool)
+  when: not (neofs_storage__use_compose | bool) and (neofs_storage__binaries_downloads | bool)
```

Comment:
suggest to use "or" here instead of "and"

Reply:
We don't need complex logic here at all. See the IR role.